app/controllers/ConfirmationController.scala

```diff
@@ -40,14 +40,24 @@ class ConfirmationController @Inject()(
                                         val controllerComponents: MessagesControllerComponents,
                                         view: ConfirmationView,
                                         errorHandler: ErrorHandler
-                                      ) extends BaseNavigationController with AuthActionHelper {
+                                      ) extends BaseNavigationController with AuthActionHelper with I18nSupport {
 
-  def onPageLoad(ern: String, arc: String, mode: Mode): Action[AnyContent] =
+  def onPageLoad(ern: String, arc: String): Action[AnyContent] =
 
     authorisedDataRequestWithCachedMovement(ern, arc) { implicit request =>
       request.userAnswers.get(ConfirmationPage) match {
         case Some(confirmationDetails: ConfirmationDetails) =>
-          logger.info(s"[onPageLoad] Successful Alert/Reject flow completed")
+          confirmationDetails.userAnswers.get(SelectAlertRejectPage).foreach {
```

Comment:
This section would be easier to read with a newline between each logging section, and the return